libpromises/evalfunction.c

```diff
@@ -3039,6 +3039,39 @@ static FnCallResult FnCallReadTcp(ARG_UNUSED EvalContext *ctx,
 
 /*********************************************************************/
 
+static FnCallResult FnCallIsConnectable(ARG_UNUSED EvalContext *ctx,
+                                        ARG_UNUSED const Policy *policy,
+                                        const FnCall *fp,
+                                        const Rlist *finalargs)
+{
+    assert(fp != NULL);
+    if (finalargs == NULL)
+    {
+        Log(LOG_LEVEL_ERR, "Function %s requires a host name, domain name or IP address as first argument",
```

Comment:
why host name and domain name? seems like just "hostname" might be OK?

Reply:
The linked documentation PR only has hostname so might as well just reduce to mentioning hostname or IP.